docs/readmes/lte/deploy_install_docker.md

```diff
@@ -79,18 +77,7 @@ Once you see the output `Reboot this machine to apply kernel settings`, reboot y
 After reboot again SSH and become root user:
 
 ```bash
-sudo su
-cd /root
-```
-
-Now you have to update the environment file for docker-compose
-
-```bash
-sed -i 's,DOCKER_REGISTRY=,DOCKER_REGISTRY=docker.artifactory.magmacore.org/,' /var/opt/magma/docker/.env
-sed -i 's/IMAGE_VERSION=latest/IMAGE_VERSION=1.8.0/' /var/opt/magma/docker/.env
-
-# Update this only for ARM node
-sed -i 's/COMPOSE_PROJECT_NAME=agw/COMPOSE_PROJECT_NAME=agw_arm64/' /var/opt/magma/docker/.env
+sudo su - root
```

Comment:
`sudo su - root` is equivalent to `sudo -i` which is recomended AFAIK.

Reply:
Instead of becoming root, you can replace the cat below with:
```
cat << EOF | sudo tee /var/opt/magma/configs/control_proxy.yml
```

Reply:
Updated as per your suggestion.